cumulusci.yml

```diff
@@ -167,21 +162,6 @@ flows:
                 options:
                     unmanaged: False
 
-    deploy_unmanaged:
-        steps:
-            4:
-                # Disable uninstall_packaged_incremental
-                task: None
-
-    dev_org_namespaced:
```

Comment:
Wait - I use this one all of the time. This is the flow we call to set up a scratch org with a namespace. I don't think all of the steps are defined here because it's a core CCI flow.

Reply:
Right, many of the flows in this yml file is basically just adding more tasks or simply overriding. For example, dev_org does not even exist in this yml file but it is defined in sfdo tooling yml file.

Reply:
But why is this being removed? We're modifying step 3 to make sure to pass in namespaced_org: true to the update_admin_profile task - does that not matter? (I don't quite understand this but I also don't want the flow to suddenly break)

Reply:
I see so it is overriding the options for #3 task. I will add them back then!

Reply:
I think so but I don't know for certain... would be something that @jstvz could help explain :)